install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2-focal

```diff
@@ -0,0 +1,321 @@
+# Last Modified: Wed Oct 29 08:16:32 2014
+#include <tunables/global>
+
+/usr/sbin/apache2 {
```

Comment:
As with other files, the changes between xenial & focal apparmor profiles are negligible, mostly python3.5 vs python3.8. I'm not sure I see the value in maintaining two difference profiles when we can just add the 3.8 exemptions to the pre-existing apparmor profile. cc @emkll for input

Reply:
I agree, with the comment above if possible it would be best to preserve a single AppArmor profile for both platforms. This make the build logic easier to maintain and will allow us to better track the AppArmor profile history over time. Allowing 2 or 1 specific versions of a Python interpreter does not provide any meaningful mitigations here.

Other then Python 3.5 -> 3.8, there are two additions:

```
/opt/venvs/securedrop-app-code/pyvenv.cfg rw
/run/systemd/userdb/io.systemd.DynamicUser rw
```
Are write permissions necessary here?

There are also 2 _removals_ ,  are they intentional?
```
   /var/www/securedrop/static/i/securedrop.png r,                                                                     
   /var/www/securedrop/static/i/securedrop_small.png r, 
```